scripts/update-translations.sh

```diff
@@ -1,4 +1,5 @@
 #!/bin/bash
+mkdir -p ../src/main/js/dev/rest/localisation
 curl -X GET https://virkailija.untuvaopintopolku.fi/yki/api/localisation?lang=fi | node <<< "var o = $(cat); var ord = {}; Object.keys(o).sort().forEach(function(key) {ord[key] = o[key];}); console.log(JSON.stringify(ord, null, 2));"  > ../src/main/js/dev/rest/localisation/translations_fi.json
```

Comment:
Käännösten hakemista varten voisi luoda bash-funktion ja hakea tarvittavat käännökset sen kautta.
esim. Node osan voisi mielestäni jättää pois.

[PATH="path"
URL="url"

fetch_translations(LANG) {
   curl -H "Accept: application/json"   "$URL?lang=$LANG" -o "$PATH$LANG.json" --create-dirs
}

fetch_translations(en)](url)

Reply:
Nodella tehdään järjestäminen ja jsonin muotoilu riveittäin, joten voisi olla kätevää jättää paikalleen.